app/services/search_service.rb

```diff
@@ -219,4 +219,30 @@ def nearbyNodes(srchString)
     sresult
   end
 
+#GET X number of latest people/contributors 
+# X = srchString
+def recentPeople(srchString)
+      sresult = DocList.new
+      drupalusers = DrupalUser.select('*, MAX(node.changed) AS last_updated')
```

Comment:
is it possible to use User here and not DrupalUser? Sorry, just trying to reduce drupaluser usage as much as possible! We can `includes` drupaluser here if need be, but thinking of the minimal use of DrupalUser until we finally merge that into User.

Reply:
Okay...makes sense 👍 👍 .